docs/repo.toml

```
@@ -353,23 +353,41 @@ deps = [
 
 doxygen_input = [
     "../../cudax/include/cuda/experimental/__container/*.h",
+    "../../cudax/include/cuda/experimental/__device/*.cuh",
+    "../../cudax/include/cuda/experimental/__event/*.cuh",
+    "../../cudax/include/cuda/experimental/__hierarchy/*.cuh",
+    "../../cudax/include/cuda/experimental/__launch/*.cuh",
     "../../cudax/include/cuda/experimental/__memory_resource/*.h",
+    "../../cudax/include/cuda/experimental/__stream/*.cuh",
 ]
 
 doxygen_predefined = [
+  "__device__",
+  "__host__",
+  "__global__",
   "_CCCL_DEVICE=",
   "_CCCL_EXEC_CHECK_DISABLE=",
   "_CCCL_FORCEINLINE=",
+  "_CCCL_GLOBAL_CONSTANT=constexpr",
   "_CCCL_HOST=",
   "_CCCL_HOST_DEVICE=",
   "_CCCL_NODISCARD=[[nodiscard]]",
   "_CCCL_NODISCARD_FRIEND=",
   "_CCCL_STD_VER=2020",
   "_CUDA_VMR=cuda::mr",
   "_CUDA_VSTD=cuda::std",
-  "LIBCUDACXX_ENABLE_EXPERIMENTAL_MEMORY_RESOURCE=",
+  "_LIBCUDACXX_AND=&&",
   "_LIBCUDACXX_EAT_REST(x)=",
+  "_LIBCUDACXX_GLOBAL_CONSTANT=inline",
+  "_LIBCUDACXX_INLINE_VAR=inline",
+  "_LIBCUDACXX_REQUIRES(...)=",
+  "_LIBCUDACXX_TEMPLATE(A)=template<A>",
+  "_LIBCUDACXX_TEMPLATE(A, B)=template<A, B>",
+  "_LIBCUDACXX_TEMPLATE(A, B, C)=template<A, B, C>",
+  "_LIBCUDACXX_TEMPLATE(A, B, C, D)=template<A, B, C, D>",
+  "_LIBCUDACXX_TEMPLATE(A, B, C, D, E)=template<A, B, C, D, E>",
   "_LIBCUDACXX_TRAILING_REQUIRES(x)=-> x _LIBCUDACXX_EAT_REST",
```

I thought this could be fixed with 
```suggestion
  "_LIBCUDACXX_TEMPLATE(x)=template<x",
  "_LIBCUDACXX_REQUIRES(x)=, __enable_if_t<x, int> = 0>",
  "_LIBCUDACXX_TRAILING_REQUIRES(x)=-> x _LIBCUDACXX_EAT_REST",
```

I could swear I had it in my async resource PR

---

I might try that in a different PR. I finally got the docs to build, and I don't want to touch it again. 🙃